packages/blockchain-wallet-v4/src/utils/checks.js

```diff
@@ -0,0 +1,13 @@
+import { is } from 'ramda'
+
+/* Number */
+export const isNumber = is(Number)
```

Comment:
for the payment branch we have those things defined in the payment file. We will need to remember to use these ones too :D

Reply:
Yes that's a good idea